package/src/contexts/channelContext/ChannelContext.tsx

```diff
@@ -176,6 +177,9 @@ export type ChannelContextValue<
    * to still consider them grouped together
    */
   maxTimeBetweenGroupedMessages?: number;
+  Poll?: React.ComponentType;
```

Comment:
Do we need to have these UI customization props in the Channel context? Why not MessageInputContext/MessagesContext?

Reply:
As discussed, moving these to `MessagesContext`. Thanks !